app/src/main/res/layout/sheet_blood_pressure_entry.xml

```diff
@@ -79,6 +79,7 @@
           android:importantForAutofill="no"
```

Comment:
@rakshakhegde How is this commit different from https://github.com/simpledotorg/simple-android/pull/447/commits/ba4c01d82a5269897f46a7ca522b66e3c89a7057?